ROADMAP.md

```diff
@@ -2,40 +2,47 @@
 
 ## User facing roadmap
 
-### V1beta1 support and GA
+### pipeline API v1 support
 
-- Add support for v1beta1 and later on GA
-- Warn usage of PipelineResource in v1beta1 (no error but warn that it is not supported aka will work but can be broken in a later release)
-- Support for workspace, result, …
+- Add support for v1 API when it will be publish
+- Support for fields, …
 
-### Improve e2e Testing Coverage for CLI
+### triggers API v1beta1 support
 
-- We should do an assessment of what scenarios we have covered for e2e testing and figure out where additional testing is needed
+- Add support for v1beta1 triggers API
 
-### Consistency across all CLI resources for list, delete, and describe subcommands
+### results integration
 
-- We should work to make sure each resource introduced to the CLI always features these three subcommands
+Evaluate and provide an integration with the
+[results](https://github.com/tektoncd/results) project.
 
-### Filtering for list and delete commands
+### chains integration
 
-- We should provide filtering solutions for all list/delete commands associated with CLI
+Evaluate and provide an integration with the
+[chains](https://github.com/tektoncd/chains) project.
+
+### operator integration
+
+Evaluate and provide an integration with the
+[operator](https://github.com/tektoncd/operator) project.
+
+### Integration with Hub
 
-### Define and carry out strategy around creation/deletion for the CLI:
+Update support for the hub in the CLI as the hub evolves.
 
-- There are two issues open around this:
-  - https://github.com/tektoncd/cli/issues/574
-  - https://github.com/tektoncd/cli/issues/575
-- There have been requests for a create command for the CLI similar to kubectl create/apply
-- We should support one of the above strategies, or we will not support creation/updating like kubectl create/apply -f moving forward
-- If we do not support a -f option, we should remove the create -f commands from pipeline/task/resource
+### OCI Bundle support
 
-### Catalog integration
+Implement
+[TEP-0031](https://github.com/tektoncd/community/blob/main/teps/0031-tekton-bundles-cli.md)
+that propose to add support for managing Tekton OCI bundles through =.
 
-- Related to the OCI experiment
-  ([#2137](https://github.com/tektoncd/pipeline/issues/2137)) and the
-  catalog / hub, integrate better with the/a catalog
-  - Allow searching catalog for a Task, Pipeline, …
-  - Allow importing a task from the catalog into the cluster
+### Consistency across all CLI resources for list, delete, and describe subcommands
+
+- We should work to make sure each resource introduced to the CLI always features these three subcommands
```

Comment:
Tracking bug for this?

Reply:
and we may want how we want to specify how to handle breaking compatibility with switches if we start a large refactor to address consistencies.